core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidatorAndUpdater.java

```diff
@@ -246,6 +249,10 @@ private void validateVolumeCount(VolumeTemplate value, VmType vmType, VolumePara
         }
     }
 
+    private boolean isSdxComputeInstanceHostGroup(InstanceGroup instanceGroup) {
+        return SDX_COMPUTE_INSTANCES.contains(instanceGroup.getGroupName().toLowerCase(Locale.ROOT));
```

Comment:
Why not Locale.ENGLISH?

Reply:
I've seen ROOT has been used more widely in the code rather than English.

Also based on javadoc:
Useful constant for the root locale. The root locale is the locale whose language, country, and variant are empty ("") strings. This is regarded as the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.